### PR TITLE
chore: Updating default Windows VHDs for to include 11B HNS fixes

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

**Reason for Change**:

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [ ] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/api/azenvtypes.go [hunk 1]
```diff
@@ -154,26 +154,26 @@ var (
...
+		ImageVersion:   "17763.3650.221213",
 	}
 
 	// WindowsServer2019OSImageConfig is the 'vanilla' Windows Server 2019 image
 	WindowsServer2019OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "WindowsServer",
 		ImageSku:       "2019-Datacenter-Core-with-Containers-smalldisk",
 		ImagePublisher: "MicrosoftWindowsServer",
-		ImageVersion:   "17763.3287.220806",
+		ImageVersion:   "17763.3650.221105",
```
Comment (jackfrancis) [inline pkg/api/azenvtypes.go:176]: Should this be 221213?

Comment (marosset) [reply pkg/api/azenvtypes.go:176]: That VHD was published by the Windows team and was used the base for the aks-engine VHDs so it'll have a different publication timestamp.

Here's where we feed the image details into packer

https://github.com/Azure/aks-engine/blob/d22c9790163bd7a317faefbddaa6392bcedac08c/vhd/packer/windows-2019-vars.json#L1-L7

Comment (jackfrancis) [reply pkg/api/azenvtypes.go:176]: cool, just checking! :)

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Pull request merged**